cognite_toolkit/_cdf_tk/commands/_base.py

```diff
@@ -81,12 +83,14 @@ def _load_files(
             try:
                 resource = loader.load_resource(filepath, ToolGlobals, skip_validation)
             except KeyError as e:
+                print(Panel(traceback.format_exc(), title="Traceback", expand=False))
                 # KeyError means that we are missing a required field in the yaml file.
                 raise ToolkitRequiredValueError(
                     f"Failed to load {filepath.name} with {loader.display_name}. Missing required field: {e}."
                     f"\nPlease compare with the API specification at {loader.doc_url()}."
                 )
             except TypeError as e:
+                print(Panel(traceback.format_exc(), title="Traceback", expand=False))
```

Comment:
It is very hard to understand what went wrong in this cases, traceback helps a lot.